packages/swing-store/src/transcriptStore.js

```diff
@@ -252,10 +253,18 @@ export function makeTranscriptStore(
     ensureTxn();
     const { hash, startPos, endPos, incarnation } = getCurrentSpanBounds(vatID);
     const rec = spanRec(vatID, startPos, endPos, hash, 0, incarnation);
```

Comment:
For consistency with `stopUsingTranscript`:
```suggestion
    const rec = spanRec(vatID, startPos, endPos, hash, false, incarnation);
```

Reply:
yeah, probably a good idea.. there are four places where we're calling with an integer, one with a boolean, and one with the `isCurrent` we just grabbed from the DB (which means an integer). I'll change the integers to booleans, and leave the `isCurrent`. We rely upon `spanRec()` coercing to an integer anyways, since SQL (or at least SQLite) doesn't have a BOOLEAN type.